### DISCUSSION TITLE
Just curious, why can't the PWM driver chip panel use this library, and is it possible to use it?

### DISCUSSION THREAD

**Discussion by planevina** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#684] [General]
This library is a fantastic matrix LED driver library, I've made several small toys with it, thank you for your contributions.
![67bd24ac-bd9c-4308-97e2-5fbbb110fe15](https://github.com/user-attachments/assets/060e241b-fdb2-41f7-a624-9cc00caad17e)

 Recently, I bought a few 128x64 LED panels from a second-hand market. I didn't look closely when I bought them and only discovered when they arrived that the IC is 9935S+9736CP. After checking, I found out that it is a PWM-driven chip, by "ledsic" , so naturally, it can't be used.

![cd0b05dc-49a4-452b-be94-ef70b4d78342](https://github.com/user-attachments/assets/2684736b-d351-4574-9a7c-bb7e17b018d4)
I'm just curious, why can't chips like these be supported? Is it because of technical difficulties? Thank you.

**Comment by board707** [comment]
> why can't chips like these be supported?

because the lacking the info about driver. The datasheets of most of PWM type drivers are unavailable for public, these are the commercial secret of the manufacturer companies.

**Reply by planevina** [reply]
I see. Thank you for your reply
